mail_tracking/models/mail_message.py

```diff
@@ -58,20 +62,41 @@ def tracking_status(self):
             for partner in partners:
                 # If there is partners not included, then status is 'unknown'
                 partner_trackings.append((
-                    'unknown', False, partner.name, partner.id))
+                    'unknown', False, partner.name, partner.id, partner.email))
             res[message.id] = partner_trackings
         return res
 
+    def _get_email_cc(self):
+        res = {}
+        ResPartnerObj = self.env['res.partner']
+        for message in self:
+            email_cc = (message.email_cc or '').split(COMMASPACE)
+            email_cc_no_partner = []
+            groups = ResPartnerObj.read_group([('email', 'in', email_cc)],
```

Comment:
Isn't any way to already obtain this information from upstream code?

Reply:
I don't see any stuff that can help here...

Reply:
OK, maybe others that know more about the module can say.